### PR TITLE
Change debugger context menu styles to be closer to native OSX styling.

### PR DESCRIPTION
**Pull request by gavlu**
Associated Issue: #1102 

@jasonLaster 

### Summary of Changes
* Changed the background color of the context menu, drop shadow color, and `menuitem` hover color.
* Changed the padding of the `menuitem` class.
* Changed the font sizing for the context menu.

### Testing

* [x] passes `npm test`
* [x] passes `npm run lint`

### Screenshots/Videos
screen shot 2016-11-08 at 6 57 33 pm

### CODE REVIEW

**Comment by gavlu** [comment]
@jasonLaster

File: public/js/components/App.css [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 :root.theme-light,
 :root .theme-light {
-  --theme-search-overlays-semitransparent: rgba(221, 225, 228, 0.66);
```
Comment (jasonLaster) [inline public/js/components/App.css]: hmm, i don't think we'll want to change this. It's a global that's shared in many places.

I think we can likely just use ` rgba(190, 190, 190, 0.8);` for the box-shadow. the menu is special in that it's only for the local environment and...

**Comment by jasonLaster** [comment]
@gavlu this one small fix for App.css and it's ready to go

File: public/js/components/App.css (hunk 1 above)
Comment (rheesneb) [inline public/js/components/App.css]: Not sure

**Pull request merged**